app/main/views/briefs.py

```diff
@@ -151,6 +151,43 @@ def update_brief_add_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['PUT'])
+def update_brief_add_rfx_seller(brief_id, supplier_code):
+    updater_json = validate_and_return_updater_request()
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    # Find the supplier that matches the inputted code, otherwise 404.
+    supplier = Supplier.query.filter(
```

Comment:
Please use the services classes for interactions with the db.

Also it is very easy to load too much data when querying like this.  It is best to use .options(noload('*') or joinedload('table'))

Reply:
This has been updated in my latest commit.